src/components/atoms/Label/Label.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// I N T E R F A C E S
+export interface LabelProps {
+  className?: string
+  style?: React.CSSProperties
+  type?: LabelType
+}
+
+// T Y P E S
+export type LabelType =
+  'primary'
+  | 'success'
+  | 'notice'
+  | 'warning'
+  | 'error'
+  | 'white'
+  | 'light'
+  | 'dark'
+  | 'black'
+
+// C O M P O N E N T
+export const Label: React.FC<LabelProps> = ({
+  children,
+  className,
+  style,
+  type = 'primary'
+}) => {
+  return (
+    <StyledLabel
+      className={`label ${className} ${type}`}
+      style={style}
+    >
+      {children}
+    </StyledLabel>
+  )
+}
+
+// S T Y L E S
+const StyledLabel = styled.span`
+  z-index: 10;
+  margin-left: 10px;
+  padding: 4px 8px;
```

Comment:
Is it possible to use css variables here? 🤔  Or maybe relative sizes with `em`?

Reply:
variables would be possible, but i should wait for the refactoring for the space-variables in CHAP-925